modules/ROOT/pages/getting-started.adoc

```diff
@@ -11,61 +11,73 @@ NOTE: For more information on configuration, refer to the documentation for xref
 ----
 
 While the AWS documentation mentions cloud-init and scripts, FCOS does not support cloud-init or the ability to run scripts from user-data. It accepts only Ignition configuration files.
 
-=== Launching with QEMU
-. Go to the https://getfedora.org/coreos/download/[download page] to retrieve the latest image suitable for QEMU.
+=== Launching with QEMU or libvirt
+. https://getfedora.org/coreos/download/[Download and verify] the latest image suitable for QEMU.
 
 . Uncompress the file. Assuming that the downloaded file is called `fedora-coreos-qemu.qcow2.xz`, the command would be the following:
 +
-`unxz fedora-coreos-qemu.qcow2.xz`
-+
-. Ensure the correct ownership of the file:
-+
-`chown core:kvm fedora-coreos-qemu.qcow2`
+[source, bash]
+----
+unxz fedora-coreos-qemu.qcow2.xz
+----
 +
-TIP: The above example assumes that you have created the user named `core` on your local machine. If you did not, replace `core` with the user that will boot FCOS on the local host.
+TIP: Make sure that your user has access to `/dev/kvm` (this should already be the case on modern distros), or if `/dev/kvm` is owned by the `kvm` group, adding yourself to that group with `usermod -aG kvm $USER`.
 
-. Specify the Ignition file to the hypervisor. In the case of QEMU, this is done with the `-fw_cfg` parameter, which sets the `opt/com.coreos/config` key in the QEMU firmware configuration device.
+. Launch the VM using `qemu-kvm` or `virt-install`. The Ignition file is passed to the VM via the `-fw_cfg` parameter, which sets the `opt/com.coreos/config` key in the QEMU firmware configuration device. For libvirt, you must pass this through the `--qemu-commandline` argument.
 +
-You must use absolute file paths here. QEMU cannot resolve relative file paths for these options.
+.Example launching FCOS with QEMU (no persistent storage)
+[source, bash]
+----
+qemu-kvm -m 2048 -cpu host -nographic -snapshot \
+	-drive if=virtio,file=fedora-coreos-qemu.qcow2 \
+	-fw_cfg name=opt/com.coreos/config,file=path/to/example.ign
+----
 +
-.Example launching Fedora CoreOS with QEMU
+.Example launching FCOS with QEMU (persistent storage)
 [source, bash]
 ----
-qemu-system-x86_64 -machine accel=kvm -m 2048 -cpu host -nographic \
-	-drive if=virtio,file=path/to/fedora-coreos-qemu.qcow2 \
-	-device virtio-rng-pci \
+qemu-img create qcow2 -b fedora-coreos-qemu.qcow2 my-fcos-vm.qcow2
+qemu-kvm -m 2048 -cpu host -nographic \
+	-drive if=virtio,file=my-fcos-vm.qcow2 \
 	-fw_cfg name=opt/com.coreos/config,file=path/to/example.ign
 ----
 +
 .Example launching FCOS with virt-install
 [source, bash]
 ----
-virt-install -n fcos --vcpus 2 -r 2048 --os-variant=fedora31 --import --network bridge=virbr0 --disk=/var/lib/libvirt/images/fedora-coreos-30.20190905.0-qemu.qcow2,format=qcow2,bus=virtio --noautoconsole --qemu-commandline="-fw_cfg name=opt/com.coreos/config,file=/path/to/example.ign"
+virt-install -n fcos -r 2048 --os-variant=fedora31 --import --noautoconsole \
```

Comment:
let's remove `--noautoconsole` and add `--graphics=none` so we will get attached to a serial console

Reply:
I personally rarely not use `--noautoconsole` since for libvirt VMs you usually don't authenticate over serial with a password, but via SSH. Though it's useful for FCOS since it prints the IP address for you.